lab_3/start.py

```diff
@@ -119,9 +119,40 @@ def score_10(k=1000, trie_levels=(2,)):
 
     ACTUAL_6 = score_6()
     EXPECTED_DISTANCE_TO_EN_DE_PROFILES = 17, 25
+<<<<<<< HEAD
```

Comment:
Remove these lines - they are not valid Python code.

Reply:
Removed